### PR TITLE
Add a step to prepare nodes for cert rotation

### PR DESCRIPTION
**Pull request by vrutkovs**
After installer is complete nodes are still using short lived certificates, which are rotated after 24 hours. This step rotates certificates on master so that nodes could be shut down immediately

### CODE REVIEW

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-30d
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-suspend-30d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-30d
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-suspend-30d
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-60d
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-suspend-60d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-30d
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-suspend-30d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-30d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-suspend-90d
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-30d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-30d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-suspend-90d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-30d
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-suspend-90d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-60d
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-suspend-60d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-30d
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-suspend-30d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-30d periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-suspend-30d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-30d periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-suspend-30d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-30d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-30d periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-suspend-30d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-30d periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-suspend-30d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-60d /pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-suspend-60d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-60d periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-suspend-60d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-30d periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-suspend-30d periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-60d periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-suspend-60d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-30d periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-suspend-30d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-30d periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-suspend-30d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-30d periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-suspend-30d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-shutdown-30d periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-suspend-30d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ovn-ha-cert-rotation-suspend-30d

**Review by dinhxuanvu — CHANGES_REQUESTED** [review]
Looking good. One nit.

File: ci-operator/step-registry/openshift/e2e/cert-rotation/prepare-for-shutdown/openshift-e2e-cert-rotation-prepare-for-shutdown-commands.sh [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+oc -n openshift-machine-config-operator create serviceaccount kubelet-bootstrap-cred-manager
+oc -n openshift-machine-config-operator adm policy add-cluster-role-to-user cluster-admin -z kubelet-bootstrap-cred-manager
+cat << 'EOZ' > /tmp/kubelet-bootstrap-cred-manager-ds.yaml
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kubelet-bootstrap-cred-manager
+  namespace: openshift-machine-config-operator
+  labels:
+    k8s-app: kubelet-bootrap-cred-manager
```
[the remark this answers is not in the document]
Comment (vrutkovs) [reply ci-operator/step-registry/openshift/e2e/cert-rotation/prepare-for-shutdown/openshift-e2e-cert-rotation-prepare-for-shutdown-commands.sh:39]: Fixed

**Review by dinhxuanvu — APPROVED** [review]
/lgtm

**Pull request merged**